src/main/kotlin/org/rust/ide/intentions/addFmtStringArgument/AddFmtStringArgumentIntention.kt

```diff
@@ -25,6 +26,9 @@ class AddFmtStringArgumentIntention : RsElementBaseIntentionAction<AddFmtStringA
     override fun getText(): String = "Add format string argument"
     override fun getFamilyName(): String = text
 
+    override fun startInWriteAction(): Boolean = false
```

Comment:
But it seems our tests think that this intention should have a preview but actually it won't be shown and even doesn't have a lot of sense since result heavily depends on user input. Maybe disable it manually and adjust tests?

Reply:
I added

```kotlin
protected open val previewExpected: Boolean get() = intention.startInWriteAction()
```

to the tests, so intention will not have preview and tests know it

Reply:
I still think it's a good idea to provide `IntentionPreviewInfo.EMPTY` manually because otherwise we will depend on some implementation details which can lead to broken tests/unexpected behaviour in the future